### PR TITLE
Adding a question on IP address classes

### PR DESCRIPTION
**Pull request by MadhulikaSingh2304**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/information-technology-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const informationTechnologyQuiz = [
+  {
+    Question: "For a Class A IP address which octet represents the network ID?",
```
Comment (jdwilkin4) [inline src/data/information-technology-quiz.ts:3]:
```suggestion
    Question: "For a Class A IP address, which octet represents the network ID?",
```

File: src/data/information-technology-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+    Answer: "First octet",
+    Distractor1: "Second octet",
+    Distractor2: "Third octet",
+    Distractor3: "Fourth octet",
+    Explanation: "For Class A IP addresses,  the first octet (8 bits / 1 byte) represent the network ID, and the remaining three octets (24 bits / 3 bytes) are the host ID.",
```
Comment (jdwilkin4) [inline src/data/information-technology-quiz.ts:8]:
```suggestion
    Explanation: "For Class A IP addresses,  the first octet (8 bits / 1 byte) represents the network ID, and the remaining three octets (24 bits / 3 bytes) represent the host ID.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by MadhulikaSingh2304** [comment]
> Thank you for your contribution!
> 
> Please review the requested changes. Once those changes have been implemented, then we can merge your PR into main. 👍

Done, Please check.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**